src/core/server/base.js

```diff
@@ -336,6 +340,16 @@ function baseServer(
           store.dispatch(dismissSurvey());
         }
 
+        const mozClientIdHeader = DISCO_TAAR_CLIENT_ID_HEADER;
```

Comment:
I suppose you can use the constant directly now, no need to create a variable here.